### ISSUE TITLE
Feature Request: Provide a way to ignore packages

### ISSUE THREAD

**Issue by billdenney** [DylanDijk/CTVsuggest#2]
I just ran `CTVsuggest(taskview = "Pharmacokinetics", n = 20)`, and it came up with a list of several useful packages.  It also is suggesting some packages that are unrelated (no surprise and no issue there).

I wonder if there could be some method of removing packages that have been evaluated and are not of interest from the suggested list.

My initial thought of a way to do this would be to put the list in the yaml header of the CTV source .md file like

```
CTVsuggest_ignore: ['package1', 'package2', 'package3']
```

But, I'm sure that there are many other ways that could also work.

**Comment by DylanDijk** [comment]
Adding the packages, that have been evaluated and are not of interest, somewhere in the source .md file is a good idea and would work well. I read in the .md file as part of the model training, and therefore this should be straightforward to implement. 

We would then just need to announce this to the rest of the maintainers.

**Comment by DylanDijk** [comment]
Hi, in the end it was decided that we should not put ignored packages in the TV source file, but instead to have an additional `ignore` argument for `CTVsuggest()`.

With a new install, you can now write:

```
CTVsuggest::CTVsuggest("Econometrics", ignore = c(
  "GVARX" # better suited for TimeSeries TV
 ))
```
You should get the following output:
```
            Econometrics  Packages
JFE          0.9982844       JFE
iClick       0.9945763    iClick
lboxcox      0.9939188   lboxcox
wildrwolf    0.9934578 wildrwolf
gbmt         0.9932194      gbmt
```
***
Then for the automatic issue generation, I think this something that will be looked at later when more maintainers start to use `CTVsuggest`. But in the mean time I can start working on it, so that when a decision is made about where the threshold information is stored, it can be implemented.

**Comment by billdenney** [comment]
That makes sense.  In that case, I'd suggest looking into a way to allow the maintainer to run a GitHub action on their repository on a schedule that would raise an error if there were un-addressed, suggested packages.

**Comment by DylanDijk** [comment]
Yes, that makes sense thanks, the threshold can then be set in the action .yaml file.
